### PR TITLE
Add ESR versions to getSupportedVersions function result

### PR DESCRIPTION
**Pull request by sradevski**
Change-type: minor
Signed-off-by: Stevche Radevski <stevche@balena.io>

---
##### Contributor checklist

- [ ] Includes tests
- [ ] Includes typings
- [ ] Includes updated documentation
- [ ] Includes updated build output

### CODE REVIEW

File: tests/integration/models/os.spec.coffee [hunk 1]
```diff
@@ -235,6 +241,15 @@ describe 'OS model', ->
 				promise = balena.models.os.getSupportedVersions('foo-bar-baz')
 				m.chai.expect(promise).to.be.rejectedWith('No such device type')
 
+		describe 'given a user that supports ESR', ->
+			# TODO: This should be production plan user
```
Comment (sradevski) [inline tests/integration/models/os.spec.coffee]: @thgreasi there is also a paid user envvar, but for ESR we only need production or enterprise plan users. shall we create another instance of a paid user, or shall we make sure that the paid user passed is actually on a production/enterprise plan?

Comment (thgreasi) [reply tests/integration/models/os.spec.coffee]: @sradevski The paid user that we currently have is for a $0 test plan. Having a test user on a production/enterprice plan probably needs some discussion about the analytics & billing implications.

We could definitely have such a user on staging and run the respective tests when the appropriate env vars are available.

Comment (sradevski) [reply tests/integration/models/os.spec.coffee]: @thgreasi if we can enable ESR for the test plan, then that would work as well. Is that possible?

Comment (thgreasi) [reply tests/integration/models/os.spec.coffee]: I haven't seen the API code for the ESR feature, but that should work as well.

**Comment by sradevski** [comment]
Closing this as the implementation details completely changed. Will reopen a new PR once the changes are final.

**Pull request closed without merging**